services/compute/virtualmachine/virtualmachine.go

```diff
@@ -457,12 +457,12 @@ func (c *client) getVirtualMachine(vm *wssdcloudcompute.VirtualMachine, group st
 		vmtype = compute.StackedControlPlane
 	}
 	return &compute.VirtualMachine{
-		Name:                     &vm.Name,
-		ID:                       &vm.Id,
-		Tags:                     getComputeTags(vm.GetTags()),
+		Name: &vm.Name,
```

Comment:
Would you mind reverting the whitespace changes? Obviously very low priority, just will make our overall PR easier to read

Reply:
I think these are done automatically by `make fmt` according to go rules so better keep it as is.

Reply:
Makes sense!